src/components/app-configure/AppUserLimits.vue

```diff
@@ -172,31 +212,39 @@ export default {
     watch(
       () => storage.value,
       () => {
-        if (storage.value.value !== "") {
+        if (isValidByteValue(storage.value)) {
```

Comment:
Are we passing the right params here? The `isValidByteValue` function is expecting an object with `value` and `unit` fields.

Reply:
Yeah so storage is a ref object with structure `{value: Number, unit: String}`. Similarly bandwidth is also a ref object with same structure